xpublish/utils/zarr.py

```diff
@@ -5,28 +5,18 @@
 import numpy as np
 import xarray as xr
 from numcodecs.compat import ensure_ndarray
-from xarray.backends.zarr import encode_zarr_variable
+from xarray.backends.zarr import (
+    DIMENSION_KEY,
+    encode_zarr_attr_value,
+    encode_zarr_variable,
+    extract_zarr_variable_encoding,
+)
 from zarr.meta import encode_fill_value
 from zarr.storage import array_meta_key, attrs_key, default_compressor, group_meta_key
 from zarr.util import normalize_shape
 
 from .api import DATASET_ID_ATTR_KEY
 
-try:
-    from xarray.backends.zarr import (
-        DIMENSION_KEY,
-        encode_zarr_attr_value,
-        extract_zarr_variable_encoding,
-    )
-except ImportError:
-    # xarray <= 0.16.1
```

Comment:
Was it intentional to remove support for older xarray versions?

Reply:
It was. https://github.com/pydata/xarray/pull/3897 fixed this upstream. Anything after 0.16 should be fine. Noticing now that we also need to bump the minimum version requirements in setup.py.